app/assets/javascripts/discourse/app/routes/password-reset.js

```diff
@@ -27,4 +27,9 @@ export default class PasswordReset extends DiscourseRoute {
       });
     }
   }
+
+  setupController(controller, model) {
+    controller.set("model", model);
+    controller.initSelectedSecondFactorMethod();
+  }
```

Comment:
Noob question: curious why this goes here since we have a controller file. Also, do we need to call `super.setupController`? 🤔

Reply:
It's not necessary, we have many examples in core where we don't do it and they don't do it in the old ember docs either https://guides.emberjs.com/v1.10.0/routing/setting-up-a-controller/ .

Looks good TGX :+1:

Reply:
Ah. Checking the documentation for 5.5:

> The model is passed from the Route to the Controller by default using the `setupController()` function.

So I assume the default implementation is exactly `controller.set("model", model);`, which means that is the same as `super.setupController()`.

Reply:
Indeed.

https://github.com/emberjs/ember.js/blob/28444d536fa20debef2a67e2f18c5eb11113a4b5/packages/%40ember/routing/route.ts#L1345-L1349